src/backend/distributed/metadata/metadata_cache.c

```diff
@@ -478,6 +477,45 @@ IsCitusTableViaCatalog(Oid relationId)
 }
 
 
+/*
+ * ReplicationModelViaCatalog gets a relationId and returns the replication
+ * model character from pg_dist_partition via reading from catalog.
+ */
+char
+ReplicationModelViaCatalog(Oid relationId)
```

Comment:
Is it weird to have a helper method that intentionally does not use the cache to be placed in `metadata_cache.c` ?

I saw `IsCitusTableViaCatalog` in the same file and thought that it should be ok. However, I can move those to another file if you have any suggestions.

Reply:
This function can be removed now

Reply:
removed it